samples/sensor/grove_temperature/prj.conf

```diff
@@ -7,4 +7,3 @@ CONFIG_GROVE=y
 CONFIG_GROVE_TEMPERATURE_SENSOR_ADC_CHANNEL=10
 CONFIG_GROVE_TEMPERATURE_SENSOR=y
 CONFIG_GROVE_TEMPERATURE_SENSOR_V1_X=y
-CONFIG_GROVE_LCD_RGB=y
```

Comment:
why is this being removed? isn't that part of the sample to display the sensor data on the LCD?

Reply:
On some boards you will need to use 2 pull-up resistors (10k Ohm) between the
SCL/SDA lines and 3.3V

Reply:
@nashif After connecting two resistors of 10k connecting to SCA and SDA lines , I can see output temperature value on LCD. I am going to push it now.